aviary/variable_info/variable_meta_data.py

```diff
@@ -6213,18 +6214,18 @@
 )
 
 add_meta_data(
-    Dynamic.Mission.ELECTRIC_ENERGY,
+    Dynamic.Mission.CUMULATIVE_ELECTRIC_ENERGY_USED,
     meta_data=_MetaData,
     historical_name={"GASP": None,
                      "FLOPS": None,
                      "LEAPS1": None
                      },
     units='kJ',
-    desc='Total amount of electric energy consumed by the vehicle across the mission',
+    desc='Total amount of electric energy consumed by the vehicle up until this point in the mission',
 )
 
 add_meta_data(
-    Dynamic.Mission.ELECTRIC_POWER,
+    Dynamic.Mission.ELECTRIC_POWER_IN,
     meta_data=_MetaData,
     historical_name={"GASP": None,
                      "FLOPS": None,
```

Comment:
Maybe change the description from:
'Current electric power consumption of the vehicle, per single instance of each engine model'
to:
'Current electric power consumption of each engine'

Reply:
Done!